components/proposal/ListEmptyState.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-  <UCard class="h-80 flex items-center justify-center text-grey-600">
+  <div class="h-80 text-center text-grey-1000">
+    <div class="flex justify-center">
+      <img src="~/assets/images/empty-blocks.svg" class="w-40" />
```

Comment:
empty-blocks.svg is a static resource; should it be served from `public` instead of `assets`? seems irrelevant but browsers can cache static resources more aggressively